src/controllers/article.controller.js

```diff
@@ -185,17 +185,25 @@ class ArticleController {
   }
 
   /**
-   *
-   *
+   * Retunrs the specific article
+   * @async
+   * @param {object} req - Request object
+   * @param {object} res - Response object
+   * @return {json} Returns json object
    * @static
-   * @param {*} request - - express request parameter
-   * @param {*} response - - express response parameter
-   * @returns {object} - - article object
-   * @memberof ArticleController
    */
-  static async fetchSpecificArticle(request, response) {
-    const { article } = request;
-    return responseGenerator.sendSuccess(response, 200, article);
+  static async fetchSpecificArticle(req, res) {
+    const { articleId } = req.params;
+
+    const theArticle = await BaseRepository.findAverage(articleId);
+    if (theArticle.length < 1) {
+      return responseGenerator.sendError(
+        res,
+        404,
+        'The requested article was not found'
+      );
+    }
+    return responseGenerator.sendSuccess(res, 200, theArticle);
```

Comment:
Good work @tolumide-ng 👍 

However, the modification here requires that you use a try-catch block to catch server error and then write a test case for it.